app/domain/operations/product_selection_effects/dchbx_product_selection_effects.rb

```diff
@@ -81,11 +81,9 @@ def renew_enrollments(enrollment)
       end
 
       def enrollment_effective_on_eligible_for_renewal?(enrollment, bcp)
```

Comment:
@saipraveen18 Refactor the method to `enrollment.effective_on.year == bcp.start_on.year.pred`